src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.ts

```diff
@@ -35,11 +33,19 @@ export class DotEditPageMainComponent implements OnInit, OnDestroy {
         this.destroy$.complete();
     }
 
-    private subscribeIframeCloseAction(): void {
-        this.dotContentletEditorService.close$.pipe(takeUntil(this.destroy$)).subscribe(() => {
-            this.dotPageStateService.reload(this.route.snapshot.queryParams.url);
-        });
+    /**
+     * Call reload method to refresh page based on url
+     *
+     * @param {any} event
+     * @memberof DotEditPageMainComponent
+     */
+    onCustomEvent(event: any): void {
+        if (event.detail.name === 'close' && event.detail.htmlUrl) {
+            this.dotPageStateService.reload(event.detail.htmlUrl.split('?')[0]);
```

Comment:
By doing just this, you're not changing the url in the address bar,.

Reply:
yes it is changing
https://monosnap.com/file/ZXZdr7WAapC56YfgqwEZjtLyJkjzxf